recipes/trinotate/meta.yaml

```diff
@@ -0,0 +1,42 @@
+about:
+  home: https://trinotate.github.io/
+  license: ''
+  summary: 'Trinotate is a comprehensive annotation suite designed for automatic
+    functional annotation of transcriptomes, particularly de novo assembled
+    transcriptomes, from model or non-model organisms'
+
+build:
+  number: 1
+
+package:
+  name: trinotate
+  version: '3.2.2'
+
+requirements:
+  build:
+      - perl-app-cpanminus
+      - perl-threaded
+      - perl-module-build
+      - perl-dbi
+      - perl-file-find-rule
+  run:
+      - libgcc
```

Comment:
libgcc can be remove.
More for information please look at: https://github.com/bioconda/bioconda-recipes/blob/master/GUIDELINES.md#cc